lib/gtfsplus/components/GtfsPlusVersionSummary.js

```diff
@@ -1,16 +1,31 @@
 // @flow
 
+import moment from 'moment'
 import React, {Component} from 'react'
-import {Panel, Row, Col, Table, Button, ButtonToolbar, Glyphicon, Alert} from 'react-bootstrap'
+import {
+  Alert,
+  Button,
+  ButtonToolbar,
+  Col,
+  Glyphicon,
+  Panel,
+  Row,
+  Table
+} from 'react-bootstrap'
 import {browserHistory, Link} from 'react-router'
-import moment from 'moment'
 
-import {getGtfsPlusSpec} from '../../common/util/config'
 import * as gtfsPlusActions from '../actions/gtfsplus'
-
+import {getGtfsPlusSpec} from '../../common/util/config'
```

Comment:
Can you put a new line between this import and the type imports? We like to keep those separate.